2_process.R

```diff
@@ -9,6 +9,8 @@ source("2_process/src/calc_daily_light.R")
 source("2_process/src/metab_utils.R")
 source("1_fetch/src/write_data.R")
 
+library(sf)
```

Comment:
```suggestion
# Explicitly attach sf package to handle geometry data when mapping over `p1_reaches_sf`
library(sf)
```

Reply:
^ A small suggestion to remind ourselves why the call to `library(sf)` is needed here. Also, do you think it makes sense to move this line to `_targets.R`? I think I'm more likely to see it there, and that would keep our config-type stuff in one place.